src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package duke;
+
+import java.text.ParseException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Dealine Class to represent a class that will store Deadline object that extends from Tasks Class
+ * @author amresh A0235398R
+ */
+public class Deadline extends Task {
```

Comment:
+1 to [Lezheng's comment](https://github.com/nus-cs2103-AY2223S1/ip/pull/237/files#r959589248), but perhaps you could name that folder "task" instead of "Command". The latter would be an appropriate folder to create if you had classes like `AddCommand`, `DeleteCommand` and other `Command` abstractions which you did not implement (and that's fine).